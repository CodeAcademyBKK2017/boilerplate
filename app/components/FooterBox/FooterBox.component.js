import React, {Component} from 'react';
import styles from './FooterBox.component.style';
import {
  Text,
  View
} from 'react-native';

export default class Footer extends Component {
  render () {
    return (
      <View style={styles.footerStyle}>
        <Text style={styles.textFooter}>About Us</Text>
      </View>
    );
  }
}