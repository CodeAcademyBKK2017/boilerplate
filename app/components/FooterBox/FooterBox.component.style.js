import {StyleSheet} from 'react-native';

export default StyleSheet.create({
  footerStyle: {
    display: 'flex',
    width: '100%',
    height: 50,
    backgroundColor: 'white',
    justifyContent: 'center',
    bottom: 0
  },
  textFooter: {
    textAlign: 'center',
    fontWeight: 'bold'
  }
});
