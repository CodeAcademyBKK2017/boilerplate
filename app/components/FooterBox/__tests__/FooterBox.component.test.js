import 'react-native';
import FooterBox from '../FooterBox.component';
import React from 'react';
import renderer from 'react-test-renderer';

describe('TextArea', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <FooterBox />
    );
    expect(tree).toBeDefined();
  });

  it('TextArea: Should return TextArea', () => { // example to test class methods
    const tree = renderer.create(<FooterBox />);
    expect(tree).toMatchSnapshot();
  });
});