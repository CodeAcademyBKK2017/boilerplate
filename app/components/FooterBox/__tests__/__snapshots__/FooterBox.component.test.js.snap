// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextArea TextArea: Should return TextArea 1`] = `
<View
  style={
    Object {
      "backgroundColor": "white",
      "bottom": 0,
      "display": "flex",
      "height": 50,
      "justifyContent": "center",
      "width": "100%",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontWeight": "bold",
        "textAlign": "center",
      }
    }
  >
    About Us
  </Text>
</View>
`;
